modules/appeals_api/app/models/appeals_api/supplemental_claim.rb

```diff
@@ -42,6 +42,7 @@ def self.date_from_string(string)
       :required_claimant_data_is_present,
       :validate_claimant_type,
       :contestable_issue_dates_are_in_the_past,
+      :validate_retrieve_from,
```

Comment:
Would changing this to `:validate_retrieve_from_dates` make sense?

Reply:
I think it's reasonable to be more explicit. still a shorter name than the method right above it. I'll make that change 👍